packages/login/network/BitcoinEngine.js

```diff
@@ -145,3 +145,17 @@ export class BCCEngine extends BitcoinEngine {
     }
   }
 }
+
+export class DASHEngine extends BitcoinEngine {
+  _signInputs (txb, inputs, options) {
+    txb.enableBitcoinCash(true)
```

Comment:
I suspect that we need to find out appropriate replacement for the method enableBitcoinCash.